File: src/main/java/com/liu/guess/Game.java
package com.liu.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secert = random.nextInt(10) + 1;
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while(number != secert){
            System.out.print("Please input a number: ");
            number = scanner.nextInt();
            if (number < secert){
                System.out.print("Please higher: ");
            }else if(number > secert) {
                System.out.print("Please lowwer");
            }else {
                 System.out.print("You got it!!");

            }
//            System.out.println(number);
        }
//        int i = 0;
//        do {
//            i++;
//            System.out.println(i);
//        }while(i < 3);
    }
}

File: src/main/kotlin/com/liu/Game.kt
package com.liu

import java.util.*


fun main(args: Array<String>){
    val secret = Random().nextInt(10)+1
    println(secret)
//    val scanner = Scanner(System.`in`)
    var number = 0
    while(number != secret){
        print("Input a number: ")
        number = readLine()!!.toInt()
//        number = scanner.nextInt()
        if (number < secret){
            print("Higher!")
        } else if(number >secret){
            print("Lowwer!")
        } else{
            print("Great! the number is $number !")
        }
    }


}


class Game {

}